meson.build

```diff
@@ -116,6 +116,7 @@ hw_ip_spi_device_reg_h = gen_hw_hdr.process('hw/ip/spi_device/data/spi_device.hj
 hw_ip_rv_timer_reg_h = gen_hw_hdr.process('hw/ip/rv_timer/data/rv_timer.hjson')
 hw_ip_uart_reg_h = gen_hw_hdr.process('hw/ip/uart/data/uart.hjson')
 hw_ip_usbdev_reg_h = gen_hw_hdr.process('hw/ip/usbdev/data/usbdev.hjson')
+hw_ip_pwrmgr_reg_h = gen_hw_hdr.process('hw/ip/pwrmgr/data/pwrmgr.hjson')
```

Comment:
`pwrmgr` is among the happy modules that have become templated, so please point `hw/top_earlgrey/ip/pwrmgr/data/autogen/pwrmgr.hjson` instead